xclim/indicators/land/_streamflow.py

```diff
@@ -94,17 +67,22 @@ def cfcheck(self, **das):
     compute=rb_flashiness_index,
 )
 
-stats = Stats(
+
+stats = Streamflow(
     title="Statistic of the daily flow for a given period.",
-    identifier="stats",
+    identifier="discharge_stats",
     var_name="q{indexer}{op:r}",
     long_name="Daily flow statistics",
     description="{freq} {op} of daily flow ({indexer}).",
     units="m^3 s-1",
-    compute=declare_units(da=None)(generic.select_resample_op),
+    compute=generic.select_resample_op,
+    missing="any",
+    input={"da": "discharge"},
+    _version_deprecated="0.40",
 )
 
-fit = Fit(
+
+fit = Indicator(
     title="Distribution parameters fitted over the time dimension.",
     identifier="fit",
```

Comment:
```suggestion
    identifier="discharge_fit",
```
Or `discharge_distribution` ? We currently can't have two indicators with the same identifier, even from different realms.

Reply:
discharge_distribution_fit ?

Reply:
Also good with me. 
This will make the identifier different from the name in the `land` namespace. This is not uncommon in xclim, but annoying. Changing it would however be a worst breaking change than only changing the identifier. I'll leave the decision to you.